app/views/shared/_browse_action_link.erb

```diff
@@ -1,17 +1,6 @@
 <%= render "govuk_publishing_components/components/action_link", {
-  text: locals[:text],
-  href: locals[:href],
+  text: link[:title],
+  href: link[:url],
```

Comment:
And then we just pass in `data_attributes: link[:data_attributes],`

Reply:
Please, see my other replies.